packages/common/src/controllers/AccountController.ts

```diff
@@ -103,26 +101,6 @@ export default class AccountController {
     return this.accountService.sandbox;
   }
 
-  updatePersonalShelves = async () => {
```

Comment:
Hm, how do we update shelves now after the registration?

Reply:
That's a good question, I will test this tomorrow 😄

Reply:
I've moved the `restoreFavorites` and `restoreWatchHistory` calls to the `afterLogin` method. Before, the restore methods were called three times in a row when the session was restored. It still calls restore twice because the AppController and AccountController will call these methods. But I think this problem is already on the develop branch.

Reply:
Thank you. But do we need to call `restoreFavorites` and `restoreWatchHistory` separately in the `login` method (we also call `afterLogin`)?

Reply:
It also looks like we lose watch history now after a customer completes the registration. The use case:

1. I watch some videos and add them to Favorites (I don't have any registration yet, it can be free content).
2. I decide to register and my watchhistory gets overwritten with the data from the api (empty list).

Reply:
While registering, I also saw the the password field error was highlighted even though it matched the expected format. However I am not sure yet whether it has something to do with the changes in this PR.

Reply:
I fixed the restoration and personal shelves after registering.

I couldn't reproduce the password error 🤔 Do you have more information about this?

Reply:
Okay, I think the problem is that the error for the password field is also shown when the account already exists (though the proposed password itself was okay).

<img width="436" alt="Screenshot 2024-02-20 at 15 01 06" src="https://github.com/jwplayer/ott-web-app/assets/27254986/39980270-93d9-491c-93f5-065c9d291da7">

Reply:
Ah yes, we've also found that (a11y) issue and fixed that here: https://github.com/Videodock/ott-web-app/pull/107

This will be submitted next week 😄